comp-sci/data-structures-and-algorithms/intro-data-structures/the-linked-list-data-structure.md

```diff
@@ -85,6 +85,10 @@ Which of the following concepts is list-specific?
 [1:Abstract Data Type]
 Abstract Data Type, or ADT, is a definition of a data type (or a data structure) that only mentions what operations are to be performed but not how these operations will be implemented.
 
+A book is a good example of an Abstract Data Type. You know it's supposed to have a cover, a title, pages, and content. What you don't know is what those are.
+
+A telephone book is an implementation of the Abstract Data Type, because you now you know how the cover looks like, what the title is, and what the content is as well.
```

Comment:
```suggestion
A telephone book is an implementation of the Abstract Data Type because you know how the cover looks like, what the title is, and what the content is as well.
```

Reply:
Nice examples :+1:

Reply:
nice spot @Stefan-Stojanovic :muscle: